### PR TITLE
Visual Studio cheatsheet (reviving #1225) 

### PR DESCRIPTION
**Pull request by pavithirakc**
Please refer to #1225 for metadata.

as part of resolving  #980

---
IA Page: https://duck.co/ia/view/visual_studio_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@pavithirakc Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=visual studio cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/visual_studio_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/visual-studio.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            "val": "Disassembly",
+            "key": "[Ctrl] [Alt] [D]"
+        },
+        {
+            "val": "Apply Code Changes",
+            "key": "[Alt] [F10]"
+        },
+        {
+            "val": "Locals",
+            "key": "[Ctrl-Alt-V],[L]"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/visual-studio.json]: This one looks incorrect. Should be: `[Ctrl] [Alt] [V], [L]`

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=visual studio cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/visual_studio_cheat_sheet_mobile.png)

**Comment by pavithirakc** [comment]
@moollaza I too felt that it was incorrect,but when referring to the docs at
http://docs.duckduckhack.com/frontend-reference/cheat-sheet-reference.html

It states,
'"
Consecutive Keys (e.g., pressing A, then pressing B)

We recommend expressing consecutive key presses as separate code blocks separated by a comma, e.g. "[Ctrl-B], [x]"

"
Could you please clarify?

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=visual studio cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/visual_studio_cheat_sheet_mobile.png)

**Comment by pavithirakc** [comment]
@moollaza  I updated the incorrect key syntax.Can you please review it

**Comment by moollaza** [comment]
@pavithirakc sorry for the delay -- this one seems to have a problem on Beta right now and needs investigation. 

The code looks good to me now though :+1:

**Comment by moollaza** [comment]
@pavithirakc This LGTM :+1:

Thanks again for taking the time to contribute! We really appreciate it :smiley:

**Comment by moollaza** [comment]
@pavithirakc thanks again for creating the MS Visual Studio Cheat Sheet! You’ve done a really good job here and we'd like you to help our community tackle the next big challenge - our mission to become the best search engine for programmers: https://duck.co/blog/post/297/help-for-programmers.

When you're ready to move on, please: 

- Create a new Instant Answer to increase our Programming IA coverage

OR

- Test live programming Instant Answers (or those in development) for bugs and suggest improvements 

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage).

Thanks again!

**Pull request merged**